.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -65,73 +53,66 @@ assignees: ''
+9. - [ ] Cut a guest-components v<TARGET_RELEASE> release
 
-14. - [ ] Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
+10. - [ ] Cut a td-shim v<TARGET_RELEASE> release, if changes happened in the project
 
-15. - [ ] Update Enclave CC to use the released version of image-rs
+11. - [ ] Update Enclave CC to use the released version of image-rs
 
     * redo step 3, but now using v<TARGET_RELEASE>
 
-16. - [ ] Update Kata Containers to the latest released version of:
+12. - [ ] Update Kata Containers to the latest released version of:
 
     * image-rs (redo step 4, but now using the v<TARGET_RELEASE>)
     * attestation-agent (redo step 5, but now using the v<TARGET_RELEASE>)
     * td-shim (redo step 6, but now using the v<TARGET_RELEASE>)
 
-17. - [ ] Update the operator to use the images generated from the latest commit of both Kata Containers and Enclave CC
+13. - [ ] Update the operator to use the images generated from the latest commit of both Kata Containers and Enclave CC
 
     * redo step 8, but now targetting the latest payload image generated for Kata Containers and Enclave CC
 
-19. - [ ] Make sure all the operator tests are passing
+14. - [ ] Make sure all the operator tests are passing
 
-19. - [ ] Cut an Enclave CC release
+15. - [ ] Cut an Enclave CC release
 
-20. - [ ] Add a new Kata Containers tag
+17. - [ ] Add a new Kata Containers tag
 
 
-21. - [ ] Wait for release kata-runtime-payload to be successfully built
+18. - [ ] Wait for release kata-runtime-payload to be successfully built
     * After the Kata tag is created wait for (https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload.yaml) to be successfully completed, so the latest commit kata-runtime-payload for the release is created
 
-22. - [ ] Update peer pods to use the release versions and then cut a release following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#cutting-releases)
+19. - [ ] Update peer pods to use the release versions and then cut a release following the [documented flow](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/Release-Process.md#cutting-releases)
 
 ## Release
 
 
-23. - [ ] Update the operator to use the release tags coming from Enclave CC and Kata Containers
+20. - [ ] Update the operator to use the release tags coming from Enclave CC and Kata Containers
 
     * redo step 8, but now targeting the latest release of the payload image generated for Kata Containers eand Enclave CC
 
-24. - [ ] Update the Operator version
+21. - [ ] Update the Operator version
 
     * https://github.com/confidential-containers/operator/blob/main/config/release/kustomization.yaml#L7
 
-25. - [ ] Cut an operator release
+22. - [ ] Cut an operator release
 
-26. - [ ] Make sure to update the release notes
+23. - [ ] Make sure to update the release notes
```

Comment:
and tag the `confidential-containers` repository

Reply:
Done.